### DISCUSSION TITLE
Issue running prisma commands with Deno

### DISCUSSION THREAD

**Discussion by riefer02** [prisma/prisma#16676] [Q&A] [answered]
Hi Prisma Team and friends,

I suddenly getting errors for my Prisma integration with my Deno project. It was working a few days ago, but when I went to pull the DB to create the schema recently or run any Prisma command it particularly I am greeted with an error.

```
% deno run -A --unstable npm:prisma generate

> Downloading Prisma engines for Node-API for darwin-arm64 [                    ] 0%Error: native promise missing, set fetch.Promise to your favorite alternative
```

```
node -v
v18.12.1
```

Let me know what you think. Cheers.

**Comment by janpio** [comment]
What version of Deno are you using?

**Reply by riefer02** [reply]
```
deno 1.25.0 (release, aarch64-apple-darwin)
v8 10.6.194.5
typescript 4.7.4
```

**Reply by riefer02** [reply]
I just ran `deno upgrade` and it is updated to the latest and the command worked flawlessly.

I had installed deno via `homebrew` so I assumed updating the version there would affect the entire system.

Thanks so much for being on top of it @janpio.

I just want to say thank you and I see you all over these forums helping! Cheers to you and the team.

**Comment by riefer02** [comment, marked as answer]
Updated my deno version with the command:

```
deno upgrade
```

Prisma running smoothly.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
